doc/examples.md

```diff
@@ -0,0 +1,4 @@
+# Examples
+
+- [ ] [[simple]]
+- [ ] [[share-services]]
```

Comment:
```suggestion
- [ ] [[simple]]#
- [ ] [[share-services]]#
```

You also want to link `[[examples]]#` from somewhere. 

Make sure no dangling pages are shown.

Reply:
I don't think we need dedicates pages for each example. But `share-services.md` seems like a right candidate for actually documenting how to share them.